Buildings/Controls/OBC/CDL/Continuous/NormalizedDelay.mo

```diff
@@ -0,0 +1,64 @@
+within Buildings.Controls.OBC.CDL.Continuous;
+block NormalizedDelay "Calculate a time delay based on a relay experiment"
+  parameter Real gamma=3 "Asymmetry level of a relay tuner";
+  Interfaces.RealInput dtON "Half-period length for the higher value"
+    annotation (Placement(transformation(extent={{-140,40},{-100,80}}), iconTransformation(extent={{-140,40},{-100,80}})));
+  Interfaces.RealInput dtOFF "Half-period length of the lower value"
+    annotation (Placement(transformation(extent={{-140,-80},{-100,-40}}), iconTransformation(extent={{-140,-80},{-100,-40}})));
+  Interfaces.RealOutput y "Normalized time delay"
+    annotation (Placement(transformation(extent={{100,-20},{140,20}})));
+  Max max annotation (Placement(transformation(extent={{-62,40},{-42,60}})));
+  Min min annotation (Placement(transformation(extent={{-60,-64},{-40,-44}})));
+  Divide rho "Half-period ratio" annotation (Placement(transformation(extent={{-16,-10},{4,10}})));
+  Modelica.Blocks.Sources.RealExpression zeroCheck(y=noEvent(if min.y > 0 then min.y else 1))
```

Comment:
This relays on inequality of a `Real` variable which is not allowed per the Modelica Language Standard.

Reply:
Removed all the RealExpression blocks